Cargo.toml

```
@@ -34,6 +35,7 @@ binary = ["xmas-elf", "x86_64", "usize_conversions", "fixedvec", "llvm-tools", "
 vga_320x200 = ["font8x8"]
 recursive_page_table = []
 map_physical_memory = []
+sse = []
```

After making the `bit_field` dependency optional, you need to change this line to

```suggestion
sse = ["bit_field"]
```